Issue #1

carlan opened the pull request:
- Added CONTRIBUTING.md
- Updated README.md to link to CONTRIBUTING.md
- Closes #1

carlan committed:
Issue #1

- Added CONTRIBUTING.md
- Updated README.md to link to CONTRIBUTING.md

ivynasantino commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,68 @@
+
+Existem algumas maneiras de contribuir com este projeto. Você pode contribuir atualizando a documentação, sugerindo melhorias ou melhorando a base de códigos. 
+
+Alguns exemplos: adicionar testes a base de códigos atual, alterar o README para incluir algo novo que você acha necessario ou sugerir uma melhoria que não se encontra no aplicativo.
```
```suggestion
Alguns exemplos: adicionar testes a base de códigos atual, alterar o [README](https://github.com/dados-congresso/raspador-discursos-camara/blob/master/README.md) para incluir algo novo que você acha necessário ou sugerir uma melhoria que não se encontra no aplicativo.
```

ivynasantino commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,68 @@
+
+Se você encontrou algum bug ou quer adicionar uma nova funcionalidade, siga as seguintes instruções.
+
+Se depois de verificar a lista das issues (abertas e fechadas), tem certeza de que se trata de algo novo ou não reportado anteriormente, crie uma nova issue no GitHub. Lembre-se de adicionar o máximo de informações relevantes o possível para que pelo menos sejamos capazes de reproduzi-lo. 
```
```suggestion
Se depois de verificar a lista das issues (abertas e fechadas), tem certeza de que se trata de algo novo ou não reportado anteriormente, crie uma nova issue no GitHub. Lembre-se de adicionar o máximo de informações relevantes o possível para que pelo menos sejamos capazes de reproduzí-lo. 
```

ivynasantino commented on README.md:
```diff
@@ -40,4 +40,4 @@ data/
 #### Como contribuir?
 
-Caso você deseje contribuir com o nosso projeto, leia o [CONTRIBUTING.md]().
+Caso você deseje contribuir com o nosso projeto, leia o [CONTRIBUTING.md](CONTRIBUTING.md).
```
```suggestion
Caso você deseje contribuir com o nosso projeto, leia o [CONTRIBUTING](CONTRIBUTING.md).
```

carlan replied:
- Removing file extension on README.md link to CONTRIBUTING suggested by ivynasantino

ivynasantino reviewed (changes requested):
Antes de tudo, obrigado pela contribuição, gostei bastante da linha de raciocínio abordada. E, apontei algumas sugestões no texto, tu poderia dar uma olhada?

carlan committed:
Update CONTRIBUTING.md

- Added link to README.md suggested by by ivynasantino

Co-Authored-By: Ivyna Santino <ivyna.alves@ccc.ufcg.edu.br>

carlan committed:
Update CONTRIBUTING.md

- Missing graphic accent suggested by by ivynasantino

Co-Authored-By: Ivyna Santino <ivyna.alves@ccc.ufcg.edu.br>

carlan committed:
Update README.md

- Removing file extension on README.md link to CONTRIBUTING suggested by ivynasantino

Co-Authored-By: Ivyna Santino <ivyna.alves@ccc.ufcg.edu.br>

carlan commented:
Ola @ivynasantino,

Obrigado pelos comentarios e sugestoes. Atualizei os arquivos conforme suas sugestoes.
